server/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const morgan = require('morgan');
+
+const app = express();
+const port = 3000;
+
+app.use(bodyParser.json());
```

Comment:
here's that reference for the built in body parser https://expressjs.com/en/4x/api.html#express.json